monitor/census.go

```diff
@@ -1976,12 +1976,20 @@ func AIRequestError(code string, pipeline string, model string, orchInfo *lpnet.
 	}
 }
 
-func AIContainersInUse(currentContainersInUse int) {
-	stats.Record(census.ctx, census.mAIContainersInUse.M(int64(currentContainersInUse)))
+func AIContainersInUse(currentContainersInUse int, model, uri string) {
```

Comment:
What about this "InUse" metric. Didn't we want to remove it as we discussed yesterday?